dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -763,8 +769,12 @@
           }
         },
         "documentType": {
+          "x-ms-enum": {
+            "name": "documentType",
+            "modelAsString": true
+          },
           "type": "string",
-          "description": "Document type that can be provided as input for Fhir Documents.",
+          "description": "Document type that can be provided as input for Fhir Documents. Expect to have fhirVersion provided when used. Behavior of using None enum is the same as not using the documentType parameter.",
```

Comment:
should None be removed then since the parameter is not required? I am not sure what is value of passing None.

Reply:
Usage pattern difference for customers who use this feature separately. They will be using the other enum types as well as None.

Reply:
Clarified offline that there can be users who use fhir and document type together, using none for them would mean to remove the fhirversion as well as the document type parameter - easier usage would be : document type none.